### ISSUE TITLE
cannot compile it.

### ISSUE THREAD

**Issue by mingyEx** [ahmetcanaydemir/Fake-Wana-Decrypt0r#3]
Open with the latest version of dotnet4.8, it will be a bug.
Go to the official website to download
https://dotnet.microsoft.com/download/dotnet-framework/

200MB+, said it is an offline download package, it will still connect to the Internet during installation and will not move for an hour...

https://social.msdn.microsoft.com/Forums/zh-CN/954947f6-1167-4994-93a1-0cdf3f1d8683/dot-net-framework-35-sp1-offline-installer?forum=netfxsetup

The same goes for others.

https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10

But when I opened it, it just didn't change for a long time...

Think about the same as the MFC platform toolset, compare the version number and so on.
Then I find:

You can download .NET Framework 3.5 from the Microsoft Download Center. (To use .NET Framework 3.5 on Windows 8, it must be enabled on the computer.
Hmm... is it because 3.5 is useless after win8, so it is changed to optional?

Anyway, download+open, right?

https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2008/zw4w595w(v=vs.90)?redirectedfrom=MSDN
https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2008/bb822049(v=vs.90)?redirectedfrom=MSDN

In short, things before 2010.
https://docs.microsoft.com/zh-cn/dotnet/csharp/whats-new/csharp-version-history
C# version 3.0 and Visual Studio 2008 were released in the second half of 2007

https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20VS2015.md
A tuo of changes.

my question here is?
why you create a project using dotnet3.5,in 2017? ?
I don't know dotnet,and I have a lot of thing to do ,so I have no time to learn about it..can you update the project so I can run it?
thanks.

**Comment by ahmetcanaydemir** [comment]
Thank you for your feedback. I upgraded to .NET Framework v4.5. I choose dotnet 3.5 in 2017 because normally older .NET Version supported by new version and this project is not need any funcy new technology. I hope it works now.
